src/components/dialogField.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Button,Dialog,DialogTitle,DialogActions,DialogContent} from "@mui/material";
+
+export default function DialogField(props) {
+  
+   
+    return (
+        <div>
+        <Button variant="contained" color="info" onClick={props.handleClick}>
```

Comment:
remove props, destruct in in arguments.
like so:


`export default function DialogField({handleOnClick, isOpen... }) { ... 
}`

make sure call it handleOnClick, handleOnContinue, handleOnClose
everything that is boolean, start calling it as "isSomething" ( isOpen / isFullWidth )